src/modules/AutoinstConfig.rb

```diff
@@ -522,6 +527,38 @@ def MainHelp
       main_help
     end
 
+    # Reports name of base product as obtained from AY profile
```

Comment:
Contradiction: a name is a String but `@return` says Y2Product

Reply:
Bonus: Y2Product does not exist. Did you mean http://www.rubydoc.info/github/yast/yast-packager/master/Y2Packager/Product ?

Reply:
yes, y2packager::product